src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -754,6 +755,14 @@ export default class ReimbursementRequestService {
       }
     });
 
+    const recipientSettings = await prisma.user_Settings.findUnique({
+      where: { userId: reimbursementRequest.recipientId }
+    });
+
+    if (!recipientSettings) throw new NotFoundException('Reimbursement Request', reimbursementRequestId);
+
+    sendReimbursementRequestDeniedNotification(recipientSettings.slackId, reimbursementRequestId);
```

Comment:
missed this before but it's important. This method is an asynchronous method, that basically means it could take an indeterminate amount of time to resolve and return. Because of that we need to do `await sendReimbursementRequesteDeniedNotification`. This is basically saying, wait until this method resolves before continuing